apps/glusterfs/app.go

```diff
@@ -403,11 +403,19 @@ func (a *App) setAdvSettings() {
 		logger.Info("Post Request Volume Options: %v", a.conf.PostReqVolumeOptions)
 		PostReqVolumeOptions = a.conf.PostReqVolumeOptions
 	}
-
 	if a.conf.ZoneChecking != "" {
 		logger.Info("Zone checking: '%v'", a.conf.ZoneChecking)
 		ZoneChecking = ZoneCheckingStrategy(a.conf.ZoneChecking)
 	}
+	if a.conf.MaxVolumesPerCluster < 0 {
+		logger.Info("Volumes per cluster limit is removed as it is set to %v", a.conf.MaxVolumesPerCluster)
+		MaxVolumesPerCluster = a.conf.MaxVolumesPerCluster
```

Comment:
Does this work the way you intend it to? Seems to me that the volume count is always going to be larger than a negative number. Wouldn't it make more sense to set this to MAX_INT or the golang equivalent?

Reply:
No it doesn't. I would prefer to abstract it out by implementing a function cluster.volumeCount() > limitOfVolumesPerCluster()

Reply:
I changed to MAX_INT32.